typpete/src/context.py

```diff
@@ -329,38 +349,39 @@ def get_context_from_children(self, context_name):
         raise NameError("Context {} is not defined".format(context_name))
 
 
-def get_unfolded_assignments(node, value, z3_t, model, solver):
-    if isinstance(node, ast.Tuple):
-        if not isinstance(value, ast.Tuple):
+    def get_unfolded_assignments(self, node, value, z3_t, model, solver, definition_linenos):
```

Comment:
This function could use a docstring. And some lines in there could use a line break somewhere.